pages/linux/systemd-delta.md

```diff
@@ -0,0 +1,20 @@
+# systemd-delta
+
+> Find overridden systemd-related configuration files.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-delta.html>.
+
+- Show all overridden configuration files:
+
+`systemd-delta`
+
+- Show only files of specific types (comma-separated list):
+
+`systemd-delta --type {{masked|equivalent|redirected|overridden|extended|unchanged}}`
+
+- Show only files whose path starts with the specified prefix (Note: a prefix is a directory containing subdirectories with systemd configuration files):
+
+`systemd-delta {{/etc|/run|/usr/lib|...}}`
+
+- Further restrict the search path by adding a suffix (the prefix is optional):
```

Comment:
```suggestion
- Further, restrict the search path by adding a suffix (the prefix is optional):
```
Minor nitpick

Reply:
I generally use proofreading tools like Grammarly, etc  to validate the PRs in this the reason for the suggestion is

> An introductory word (Unfortunately), phrase (Before the party), or clause (If you can join us) sets the stage for the sentence. Introductory clauses and long phrases must always be followed by a comma, while single words and short phrases may or may not. Use a comma when the phrase is a verbal phrase (Running away), a non-essential appositive, or an absolute phrase (The audience waiting). Also use a comma when a pause is necessary or the lack of a comma will cause confusion.

IMO, both with or without the comma works that's why I suggested it as nitpick while approving the PR.

Reply:
In this case "further" is not an introduction, it describes "restrict". That's why I don't think there should be a comma.

Reply:
> In this case "further" is not an introduction, it describes "restrict". That's why I don't think there should be a comma.

Agreed, feel free to revert the change.